### ISSUE TITLE
Switch "Refresh from FIRST" buttons for events in admin console to use new FMS API fetches

### ISSUE THREAD

**Issue by gregmarra** [the-blue-alliance/the-blue-alliance#1286]
http://www.thebluealliance.com/tasks/get/usfirst_event_details/2016/17363

But the new app dashboard says there are no errors...
![image](https://cloud.githubusercontent.com/assets/57101/11387496/fd7704dc-92df-11e5-9505-026474344020.png)

Hmmm

**Comment by phil-lopreiato** [comment]
The 500 error is probably due the the FIRST website change

Did you look in "Logging" in the pull out drawer on the left side in the console? The error should also show up there. I'm not a fan of the new app dashboard, they managed to reduce the usefullness of every screen....

**Comment by gregmarra** [comment]
Found https://console.developers.google.com/logs, which I don't think was in my navigation..

```
'NoneType' object has no attribute 'text'
Traceback (most recent call last):
  File "/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.3/webapp2.py", line 1511, in __call__
    rv = self.handle_exception(request, response, e)
  File "/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.3/webapp2.py", line 1505, in __call__
    rv = self.router.dispatch(request, response)
  File "/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.3/webapp2.py", line 1253, in default_dispatcher
    return route.handler_adapter(request, response)
  File "/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.3/webapp2.py", line 1077, in __call__
    return handler.dispatch()
  File "/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.3/webapp2.py", line 547, in dispatch
    return self.handle_exception(e, self.app.debug)
  File "/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.3/webapp2.py", line 545, in dispatch
    return method(*args, **kwargs)
  File "/base/data/home/apps/s~tbatv-prod-hrd/1-3.388761874871547643/controllers/datafeed_controller.py", line 587, in get
    event = df.getEventDetails(first_eid)
  File "/base/data/home/apps/s~tbatv-prod-hrd/1-3.388761874871547643/datafeeds/datafeed_usfirst.py", line 100, in getEventDetails
    event, _ = self.parse(url, UsfirstEventDetailsParser)
  File "/base/data/home/apps/s~tbatv-prod-hrd/1-3.388761874871547643/datafeeds/datafeed_base.py", line 33, in parse
    return parser.parse(result.content)
  File "/base/data/home/apps/s~tbatv-prod-hrd/1-3.388761874871547643/datafeeds/usfirst_event_details_parser.py", line 28, in parse
    page_title = soup.find('h1', {'id': 'thepagetitle'}).text
AttributeError: 'NoneType' object has no attribute 'text'
```

**Comment by fangeugene** [comment]
Fixed in PR #1292
